reference/api/index.rst

```diff
@@ -0,0 +1,180 @@
+
+Getting a token
+***************
+
+A client id has been created and set into the client application. The last information needed to use the API is the token.
+
+To get a valid token, the client application must send the following request:
+
+.. code-block:: bash
+
+    GET https://demo.akeneo.com/api/oauth/v1/token
+        -d "client_id"=4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s \
+        -d "client_secret"=5dyvo1z6y34so4ogkgksw88ookoows00cgoc488kcs8wk4c40s \
+        -d "grant_type"=password \
+        -d "username"=michel \
+        -d "password"=michelpwd
+
+Response example:
+
+.. code-block:: json
+
+    {
+        "access_token": "NzFiYTM4ZTEwMjcwZTcyZWIzZTA0NmY3NjE3MTIyMjM1Y2NlMmNlNWEyMTAzY2UzYmY0YWIxYmUzNTkyMDcyNQ",
+        "expires_in": 3600,
+        "token_type": "bearer",
+        "scope": null,
+        "refresh_token": "MDk2ZmIwODBkYmE3YjNjZWQ4ZTk2NTk2N2JmNjkyZDQ4NzA3YzhiZDQzMjJjODI5MmQ4ZmYxZjlkZmU1ZDNkMQ"
+    }
+
+
+- `access_token` is the information that must be included in every requests to the API for the client application to be authorized (see the example below).
+- `expires_in` is the token lifespan (in seconds).
+- `refresh_token` is a special token used only to refresh your `access_token` after it expired.
+
+.. image:: ./img/authentication.png
+
+Accessing a resource
+********************
+
+.. code-block:: bash
+
+    curl https://demo.akeneo.com/api/rest/v1/categories
+        -H "Authorization: Bearer NzFiYTM4ZTEwMjcwZTcyZWIzZTA0NmY3NjE3MTIyMjM1Y2NlMmNlNWEyMTAzY2UzYmY0YWIxYmUzNTkyMDcyNQ"
+
+Refreshing an expired token
+***************************
+
+For security reasons, access tokens have a certain lifespan. A client application using an expired token will be unable
+to request the API.
+
+After a token has expired, the client application can ask a new token by calling:
+
+.. code-block:: bash
+
+    GET https://demo.akeneo.com/api/oauth/v1/token
+        -d "client_id"=4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s \
+        -d "client_secret"=5dyvo1z6y34so4ogkgksw88ookoows00cgoc488kcs8wk4c40s \
+        -d "grant_type"=refresh_token \
+        -d "refresh_token"=MDk2ZmIwODBkYmE3YjNjZWQ4ZTk2NTk2N2JmNjkyZDQ4NzA3YzhiZDQzMjJjODI5MmQ4ZmYxZjlkZmU1ZDNkMQ
+
+The response will be the same than when asking for the initial token except it will contain a new `access_token`
+and `refresh_token`.
+
+.. image:: ./img/refreshing_access_token.png
+
+Authorization
+~~~~~~~~~~~~~
+
+Like when using the PIM through the UI, ACLs are here to define what a user can and cannot do.
+In the role form, a `Web API permissions` tab includes ACLs for the API:
+
+- Overall Web API access
+- List attributes
+- List attribute options
+- List categories
+- List families
+- Create and update attributes
+- Create and update attribute options
+- Create and update categories
+- Create and update families
+
+.. image:: ./img/acl.png
+
+.. note::
```

Comment:
Maybe a warning and more detail why we recommend that. Something as: "Accessing the API provides higher privileges to the user granted. So we strongly recommend creating one or more dedicated user for the Web API."